### ISSUE TITLE
Support React 19

### ISSUE THREAD

**Issue by mayognaise** [runjuu/column-resizer#11]
Hi 👋 

Could you update the peer dependency for React 19?
Hope it is easy fix 🙏 

https://github.com/runjuu/column-resizer/blob/main/packages/react/package.json#L51

```json
  "peerDependencies": {
    "react": "^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
  },
```

**Comment by runjuu** [comment]
Thanks for reminding me! I've just released v1.3.1 to add support for React 19

**Comment by mayognaise** [comment]
@runjuu Thank you so much!!! 🚀
